Decompiler/Decompiler.csproj

```diff
@@ -1,65 +1,18 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{5F27DC08-3AE5-495C-8445-43D4E7CABA63}</ProjectGuid>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```

Comment:
Use `TargetFramework` when you have a single target, only use `TargetFrameworks` when you have multiple.

Reply:
c o n s i s t e n c y